strings/names/posibrain.txt

```diff
@@ -21,29 +25,38 @@ HG
 HIU
 HOG
 INC
+JACQ
 JJR
 JLLO
+JLS
 JNLG
 JRD
 KOR
 KOS
+MEL
 MET
 MIW
 MOTH
 MRPR
 MSO
+NOVA
 ORNG
 OSI
 PBU
 PHL
 PKP
 RR
 RZH
+SANS
```

Comment:
![image](https://user-images.githubusercontent.com/41448081/235824807-cdd77e7c-c9c9-4049-a26e-3201975d8cee.png)

Reply:
Definitely not San1984

Reply:
yea because i only have one "s" in my name